content/account/users.textile

```diff
@@ -0,0 +1,72 @@
+---
+title: User management
+meta_description: "Learn how to manage users, user roles, and the permissions associated with each role."
+meta_keywords: "users, user management, admin, owner, permission"
+---
+
+The user that creates an Ably account is assigned the account owner role. An account owner has permission to undertake any action within an account, such as inviting new users. There are two other account roles that inherit a subset of an account owner's permissions. An account may only have a single account owner.
+
+h2(#roles). User roles
+
+Users are assigned one of three roles:
+
+* Standard
```

Comment:
I know we call it standard right now (or, at least, that's how we called it on the corresponding page in FAQs, I don't think we have used term `Standard` anywhere else), but I am wondering if we can rename it in this PR, instead of doing it in the next where we will add the `Billing` role? 

The primary reason for to use `developer` is that it actually provides more clarity and information on what the role entails. `Standard` doesn't really mean anything other than "not admin", but in reality this role has access to some technical details (i.e. API keys), that other future roles (i.e. Billing) will not have access to. Therefore, I am keen to start using `Developer` instead of `Standard`

Reply:
Yep, I can update that as I only used 'standard' to differentiate it from an admin.